docs/source/webhooks.rst

```diff
@@ -156,6 +117,47 @@ Rule:
         ref: "mypack.myaction"
         parameters:
     ...
+    
+Using a Generic Webhook
+-----------------------
+
+By default, a special-purpose webhook with the name ``st2`` is already registered. Instead
+of using ``st2.core.webhook``, it allows you to specify any trigger that is known to |st2| 
+(either by default or from a custom sensor), so you can use it to trigger rules that aren’t 
```

Comment:
Yeah, it's great to point this part out.

Maybe also a good idea to say custom sensor / trigger since since v1.6 or so we now also support triggers/ directory with trigger definitions.